src/drop-down-icon.jsx

```diff
@@ -112,7 +112,7 @@ const DropDownIcon = React.createClass({
           <div onTouchTap={this._onControlClick}>
               <FontIcon
                 className={iconClassName}
-                style={this.prepareStyles(iconStyle)}>{this.props.iconLigature}</FontIcon>
+                style={iconStyle}>{this.props.iconLigature}</FontIcon>
```

Comment:
You are right, I have overlook the fact that this is not a raw html component, it's <FontIcon /> :+1: .